support/bash_completion.sh

```diff
@@ -1,4 +1,3 @@
-# shellcheck disable=SC2086,SC2128
```

Comment:
Do we need a hab bash_completion script in the builder repo or is this just a left over from the split earlier in the year?